### PR TITLE
Add Ubuntu 15.10 Runs

### PR DESCRIPTION
**Pull request by ellismg**
These are per merge, not per PR.  Part of #5349

### CODE REVIEW

**Comment by ellismg** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -8,6 +8,7 @@ def project = GithubProject
 
 // Map of os -> osGroup.
 def osGroupMap = ['Ubuntu':'Linux',
+                  'Ubuntu15.10':'Linux',
```
Comment (stephentoub) [inline netci.groovy]: As noted on the coreclr one, at some point it'd be good to rename the existing ubuntu case.  Not blocking for this PR, though.

**Comment by stephentoub** [comment]
LGTM

File: netci.groovy (hunk 1 above)
Comment (ellismg) [inline netci.groovy]: Opened dotnet/dotnet-ci#150

**Comment by stephentoub** [comment]
@mmitche, FYI, we've been seeing a bunch of errors like this on the CentOS and OpenSUSE machines:
```
ERROR: [WS-CLEANUP] Cannot delete workspace: remote file operation failed: d:\j\workspace\dotnet_corefx\centos7.1_release_bld_prtest at hudson.remoting.Channel@2beb7820:dci-win-tbld-9: java.nio.file.FileSystemException: d:\j\workspace\dotnet_corefx\centos7.1_release_bld_prtest\msbuild.log: The process cannot access the file because it is being used by another process.
```

**Pull request merged**